scripts/verify-setup.sh

```
@@ -0,0 +1,192 @@
+echo ""
+
+# 6. Test Authentication Flow
+echo "🔐 Authentication Test"
+echo "----------------------"
+
+echo -n "Testing signup endpoint... "
+signup_response=$(curl -s -X POST http://localhost:8080/auth/signup \
+    -H "Content-Type: application/json" \
+    -d '{
+        "name": "Test User",
+        "email": "test-'$(date +%s)'@example.com",
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)